app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -55,7 +55,7 @@
 // $rem-base: 16px;
 
 // Allows the use of rem-calc() or lower-bound() in your settings
-@import 'foundation/functions';
+@import 'foundation/functions';  
```

Comment:
I'm assuming the command here is `git checkout commit_hash path/to/file.rb`, where the commit hash is a commit prior to the changes in the file?